sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1825,6 +1847,18 @@ class Analyzer(
   }
 
   /**
+   * Exclude clause does not support
+   * RowNumberLike: ROW_NUMER()
+   * RankLike: RANK(), DENSE_RANK(), CUME_DIST(), etc.
+   * OffsetWindowFunctions: Lead(), Lag()
+   * @param wf
```

Comment:
I think `@param` can be removed as there is no doc for it and it's obvious. Also for `@return`.